README.md

```diff
@@ -47,6 +47,37 @@ After that, you may run the binary as any normal program:
 ```
 ./shark-clipper-linux-x64-latest.bin
 ```
+#### Using the Mac Prebuilt Binary
+You will need to set execute permissions on the Mac prebuilt binary, e.g.:
+```
+chmod +x shark-clipper-macos-x64-latest.bin
+```
+
+Mac requires an additional security step to run the prebuilt binary.
+This step needs to be done every time a new binary is downloaded and run for the first time.
+
+1) Run the binary as any normal program:
+```
+./shark-clipper-macos-x64-latest.bin
+```
+
+2) A popup may appear noting that the file cannot be opened because Apple cannot chcck it for malicious software, click "OK" to close.
```

Comment:
Almost all of you points except this one and 6 start with a verb/action.
Do the same here, e.g. 'Click "OK" on a popup that may come up saying ...'.